poetry/installation/executor.py

```diff
@@ -671,19 +671,19 @@ def _download_link(self, operation: Union[Install, Update], link: Link) -> Link:
                 archive = self._chef.prepare(archive)
 
         if package.files:
-            archive_hash = (
-                "sha256:"
-                + FileDependency(
-                    package.name,
-                    Path(archive.path) if isinstance(archive, Link) else archive,
-                ).hash()
+            file_dependency = FileDependency(
```

Comment:
Maybe rename this to `file_dep` or even `dep` ; there is a module called `file_dependency` and I accidentally auto-imported it in my IDE when typing a similar name.